README.md

```diff
@@ -3,111 +3,60 @@
+**Inductiva API** provides dozens of open-source physical simulation packages from your laptop. Users can start simulating right away, with no hardware setup issues and no software configuration headaches. We provide a transparent way to scale your simulations to the next level with one line of code.
 
-One can **run** simulations in two ways:
-- **High-level Simulation:** We provide pre-built scenarios that define a physical model (e.g., the dam break scenario in fluid dynamics). Users are allowed to define some parameters of the scenarios, choose the method of simulation with a few parameters and accelerate their simulations with the best hardware available for them. 
-
-Example of how to run the dam break scenario:
-```python
-from inductiva import fluids
-
-scenario = fluids.scenarios.DamBreak(dimensions=(1., 0.3, 0.3))
-
-output = scenario.simulate()
-video = output.render()
-```
-
-- **Low-level Simulation:** Users familiar with the simulators can submit their previously prepared simulation configuration files. In this way, users can immediatelly take advantage of performant hardware to speed up their simulations, without having to change any of their existing simulation scripts. Moreover, users may want to execute more than just one simulation and via API they can do it in a couple of lines of code. 
-
-Example of how to run a low-level simulation:
-```python
-from inductiva import fluids
-
-simulator = fluids..simulators.DualSPHysics()
-
-output_dir = simulator.run(input_dir="FlowCylinder",
-                           sim_config_filename="CaseFlowCylinder_Re200_Def.xml",
-                           output_dir="Flow",
-                           device="gpu")
-```
-
-Find more examples of simulations at the [tutorials section](https://github.com/inductiva/inductiva/tree/main/demos).
+Whether you want to start applying simulation to solve scientific/engineering problems or you are a power user of a specific open-source simulator, **Inductiva API** is here for you. 
 
 Our goal is to provide researchers and engineers with an easy and fast way to scale their simulations and explore various designs. 
 
 
-## Simulators
-
-The simulators we provide are all open-source and have their own dedicated documentation.
-
-Currently, we have available the following simulators:
-- [SPlisHSPlasH](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH)
-- [DualSPHysics](https://github.com/DualSPHysics/DualSPHysics)
-- [OpenFOAM](https://www.openfoam.com/)
-- [SWASH](https://swash.sourceforge.io/)
-- [xBeach](https://oss.deltares.nl/web/xbeach/)
-- [GROMACS](https://www.gromacs.org/)
+## Scenarios
 
-If you would like other simulators to be added, contact us at [simulations@inductiva.ai](mailto:simulations@inductiva.ai).
+**Inductiva API** contains pre-built scenarios that define a particular physical system ready to simulate. Users can select some parameters within the scenario that configure the system, run the simulation and visualize the results.
```

Comment:
just a suggestion: **Inductiva API** contains pre-built scenarios that define physical systems of interest ready to simulate. Users can adjust parameters and configure the system according to their needs, run the simulation  using the most adequate resources and visualize the results.